index.css

```diff
@@ -1,3 +1,88 @@
 html {
-  
+  text-align: center;
+  font-family: 'Delius Unicase', cursive;
+}
+
+body {
+  background-image: url("starsbg.png");
+}
+
+/***
+TITLE AND MESSAGE
+***/
+
+h1 {
+  font-size: 4rem;
+  margin-top: 0rem;
+  margin-bottom: 0rem;
+  color: #B345B9;
+}
+
+#message {
+  font-size: 1.5rem;
+  margin-top: 0rem;
+  margin-bottom: .5rem;
+  color: #F04DB5;
+  font-weight: bold;
+}
+
+/***
+GAME BOARD
+***/
+
+/*entire table*/
+#game-board {
+  width: 40%;
+  font-size: 6rem;
+  border-collapse: collapse;
+  border-spacing: 0;
+  margin-top: 3rem;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+/*size of each space*/
+td {
+  padding: 5rem;
+  cursor: pointer;
+}
+
+/*hover img over space*/
+.space_hover {
+  background-image: url("moon.png");
+  background-size: 40%;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+/*horizontal lines*/
+#top-row, #middle-row {
+  border-bottom: .5rem solid #A908E2;
+}
+
+/*vertical lines*/
+#space-1, #space-4, #space-7, #space-2, #space-5, #space-8 {
+  border-right: .5rem solid #BB29EF;
+}
+
+/***
+SPACE MARKERS
+***/
```

Comment:
This commenting style looks helpful! Especially if you've got a long file of css!